calendar-helper/index.js

```diff
@@ -64,27 +69,50 @@ class CalendarHelper {
     this.options.ignoreWeekDays = ignoreWeekDays;
   }
 
-  setShortcutButtons(shortcutButtons) {
-    this.options.shortcutButtons = shortcutButtons;
+  setShortcutButtons(buttons) {
+    this.options.shortcutButtons.buttons = buttons;
   }
 
   setHideIgnoredWeeks(hideIgnoredWeeks) {
     this.options.hideIgnoredWeeks = hideIgnoredWeeks;
   }
 
-  addShortcutButtons(page) {
-    let menuShortcutButtons = [];
+  transformShortcutButtonCallbackAction(buttonAction) {
+    const prefix = this.options.shortcutButtons.shouldPrefixButtonCallbacks
+      ? 'calendar-telegram-date-'
+      : '';
+    return `${prefix}${buttonAction}${this.options.shortcutButtons.callbackSuffix}`;
+  }
+
+  addShortcutButtons(page, shortcutButtons, placeAtBottom) {
+    const menuShortcutButtons = [];
 
-    for (let shortcutButton of this.options.shortcutButtons) {
+    for (let shortcutButton of shortcutButtons) {
       let buttonLabel = shortcutButton.label;
       let buttonAction = shortcutButton.action;
 
       menuShortcutButtons.push(
-        Markup.button.callback(buttonLabel, buttonAction)
+        Markup.button.callback(
+          buttonLabel,
+          this.transformShortcutButtonCallbackAction(buttonAction)
+        )
       );
     }
 
-    page.push(menuShortcutButtons);
+    page[placeAtBottom ? 'push' : 'unshift'](menuShortcutButtons);
```

Comment:
Can you please replace this with an if-else? I find it more readable:
```suggestion
    if (placeAtBottom) {
        page.push(menuShortcutButtons);
    } else {
        page.unshift(menuShortcutButtons);
    }
```